private/functions/Get-XESmartTargetPath.ps1

```diff
@@ -0,0 +1,86 @@
+
+    .PARAMETER EnableException
+        By default, when something goes wrong we try to catch it, interpret it and give you a friendly warning message.
+        This avoids overwhelming you with "sea of red" exceptions, but is inconvenient because it basically disables advanced scripting.
+        Using this switch turns this "nice by default" feature off and enables you to catch exceptions with your own try/catch.
+
+    .NOTES
+        Tags: ExtendedEvent, XE, XEvent, SmartTarget
+        Author: dbatools team
+
+        Website: https://dbatools.io
+        Copyright: (c) 2018 by dbatools, licensed under MIT
+        License: MIT https://opensource.org/licenses/MIT
+
+    .OUTPUTS
+        System.String. The path to XESmartTarget.Core.dll if found, otherwise $null.
+    #>
+    [CmdletBinding()]
+    param (
+        [switch]$EnableException
+    )
+
+    # First try the dbatools data directory (installed via Install-DbaXESmartTarget)
+    $dbatoolsData = Get-DbatoolsConfigValue -FullName "Path.DbatoolsData"
+    $installedDll = Join-DbaPath -Path $dbatoolsData -ChildPath "XESmartTarget", "XESmartTarget.Core.dll"
+    if (Test-Path -Path $installedDll) {
+        Write-Message -Level Verbose -Message "Found XESmartTarget.Core.dll in dbatools data directory at: $installedDll"
+        return $installedDll
+    }
+
+    # Second, try the bundled version (legacy path for backwards compatibility)
+    $bundledDll = Join-DbaPath -Path $script:libraryroot -ChildPath "third-party", "XESmartTarget", "XESmartTarget.Core.dll"
+    if (Test-Path -Path $bundledDll) {
+        Write-Message -Level Verbose -Message "Found bundled XESmartTarget.Core.dll at: $bundledDll"
+        return $bundledDll
+    }
+
+    # Third, check if XESmartTarget is installed system-wide via Get-Command
+    try {
+        $xeCommand = Get-Command -Name "XESmartTarget" -ErrorAction SilentlyContinue
+        if ($xeCommand) {
+            # Get the directory where the executable is located
+            $xeDirectory = Split-Path -Parent $xeCommand.Source
+            $systemInstalledDll = Join-Path -Path $xeDirectory -ChildPath "XESmartTarget.Core.dll"
+
+            if (Test-Path -Path $systemInstalledDll) {
+                Write-Message -Level Verbose -Message "Found system-installed XESmartTarget.Core.dll at: $systemInstalledDll"
+                return $systemInstalledDll
+            } else {
+                Write-Message -Level Warning -Message "Found XESmartTarget executable at '$($xeCommand.Source)' but could not find XESmartTarget.Core.dll in the same directory."
+            }
+        }
+    } catch {
+        Write-Message -Level Verbose -Message "Error checking for system-installed XESmartTarget: $($_.Exception.Message)"
+    }
+
+    # If we get here, XESmartTarget.Core.dll was not found
+    $message = @"
+Could not find XESmartTarget.Core.dll. XESmartTarget is no longer bundled with dbatools.
+
+To install XESmartTarget, use:
```

Comment:
we now have either Save-DbaCommunitySoftware OR Install-DbaXESmartTarget. IMHO we should just have one, and given we're taking the path of slimming down the library with external tools I'd go to leave just Install-DbaXESmartTarget